docs/api/mount.md

```diff
@@ -192,6 +192,9 @@ Reduces the current array of nodes to a value
 #### [`.reduceRight(fn[, initialValue]) => Any`](/docs/api/ReactWrapper/reduceRight.md)
 Reduces the current array of nodes to a value, from right to left.
 
+#### [`.tap(intercepter) => ReactWrapper`](ReactWrapper/tap.md)
```

Comment:
@pbernasconi sorry to have realised this only now but the signature should be `.tap(intercepter) => Self` to match the api docs for `tap`